config/log4j.properties

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
```

Comment:
discard this change... if you need, change should go in Azure-Compute-Move/src/Services/Kafka/Broker/config/log4j.properties

Reply:
This should have already been resolved in the last iteration. closing it.